tests/MultiChannelCicadaReader_test.py

```diff
@@ -0,0 +1,67 @@
+'''
```

Comment:
you need to add this script to the test.sh file in the tests directory, so it will be picked up by the Continuous Integration.
Then make sure that the data files you are calling in this test are present: in order to limit the amount of space used, I choose a very small example https://github.com/project8/mermithid/blob/master/tests/events_000007031_katydid_v2.13.0_concat.root with only a very few number of events. Maybe you have 2/3 others examples like this for the 3 channels (for a total of a few 10s kB); then you can add them here in this tests directory and use them for the test.

Reply:
Also, a suggestion in terms of organization: you should add this unit test part (simple copy and paste) into the IO_test.py file; this guy (should) contains all the tests for the IO processors in mermithid.

Reply:
I modified the test to read the root file that is already there and merged the test to IOTest